Gemfile

```diff
@@ -72,6 +73,7 @@ gem "simple_form", "~> 5.0"
 gem "slack-notifier", "~> 2.3.2"
 gem "split", "~> 3.4.1", require: "split/dashboard"
 gem "spreadsheet", "~> 1.2.4"
+gem "unread", "~> 0.11.0"
```

Comment:
**TL;DR I know we need to get this out the door so I won't hold this PR up bc of this.**

But..

Im not really sure we get a lot of benefit out of this gem vs doing this ourselves and I am worried locking us in to it will only create problems down the road. The fact we are only _really_ using it to mark emails as read and that there is no builtin way to mark emails as unread makes me even more convinced we shouldn't add this. I don't think it would take more than an hour or two to replace it if that's the concern. 

Happy to chat more about this.

Reply:
I like your feedback and if it you want we can replace it. The criteria we will need though really fits the gem. I'll need to know who read the messages and search by that stat. To me it looked like a well maintained gem that solved a need quickly. Using gems is why devving in Ruby on Rails is 10x right?